samples/OpenCVClient/CreateMarker.java

```diff
@@ -0,0 +1,28 @@
+import org.opencv.core.*;
+import org.opencv.core.Core;
+import org.opencv.core.CvType;
+import org.opencv.core.Mat;
+import org.opencv.imgcodecs.Imgcodecs;
+import org.opencv.imgproc.Imgproc;
+import org.opencv.aruco.*;
+import org.opencv.aruco.Dictionary.*;
+
+import java.io.File;
+import java.util.Arrays;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Vector;
+
+public class CreateMarker {
+   public static void main(String[] args) {
```

Comment:
I am with you that we need another a separate application to make the markers, but the location is not ideal, it should not be a part of the client.

My idea is that we use this "MarkerGenerator" to make images that we later use with the OpenCVClient. So images should be copied to the docker image that runs the OpenCVClient, and MarkerGenerator should not a part of that docker image.